apps/dotcms-block-editor/src/app/dot-block-editor/dot-block-editor.component.ts

```diff
@@ -65,5 +68,13 @@ export class DotBlockEditorComponent implements OnInit {
                 })
             ]
         });
+
+        this.setEditorStorageData();
+    }
+
+    private setEditorStorageData() {
```

Comment:
Shouldn't we do this in a setter of the input?

Reply:
Moved to a setter.